fitness_score: 0.038536694644367993
extrinsic_matrix: [0.993022323, -0.116120957, 0.0207295287, 0.23632665, 0.116544485, 0.992977619, -0.020518966, -2.88303208, -0.0181987975, 0.0227907393, 0.999578238, -0.135353759, 0, 0, 0, 1]